receiver/qubes-video-companion

```diff
@@ -4,18 +4,44 @@
 # Licensed under the MIT License. See LICENSE file for details.
 
 [ "$DEBUG" == 1 ] && set -x
+set -u -e
 
 set -E # Enable function inheritance of traps
 trap exit ERR
+unset GETOPT_COMPATIBLE
+unsupported_getopt () {
+    echo "Your getopt(1) doesn't support quoting" >&2
+    exit 1
+}
+if getopt -T > /dev/null; then unsupported_getopt; elif [ "$?" -ne 4 ]; then unsupported_getopt; fi
+name=${0##*/}
 
 usage() {
-    echo "Usage: qubes-video-companion webcam|screenshare [destination qube]"
-} >&2
+    printf '%s: Usage: qubes-video-companion webcam|screenshare [destination qube]\n' "$name"
```

Comment:
You did changed this line in the recent push, yet you did not added the new option...